.circleci/config.yml

```diff
@@ -954,57 +981,425 @@ jobs:
+          name: Wait for the connection to be established and check IP
+          command: |
+            counter=1
+            until [ -f /tmp/openvpn.log ] && [ "$(grep -c "Initialization Sequence Completed" /tmp/openvpn.log)" != 0 ] || [ "$counter" -ge 15 ]; do
+              ((counter++))
+              echo "Attempting to connect to VPN server..."
+              sleep 1;
+            done
+
+            if [ ! -f /tmp/openvpn.log ] || (! grep -iq "Initialization Sequence Completed" /tmp/openvpn.log); then
+              printf "\nUnable to establish connection within the allocated time ---> Giving up.\n"
+              exit 1
+            else
+              printf "\nVPN connected\n"
+              printf "\nPublic IP is now %s\n" "$(curl -s http://checkip.amazonaws.com)"
+            fi
+      - run:
+          name: Clone Repositories and Install Dependencies
+          command: |
+            git clone git@github.com:AudiusProject/audius-protocol.git ~/audius-protocol
+            cd ~/audius-protocol
+            python3 -m pip install -r .circleci/triggers/requirements.txt
+      - run:
+          name: Setup SSH Config
+          command: |
+            git clone git@github.com:AudiusProject/audius-ssh.git ~/audius-ssh
+            # `AddKeysToAgent yes` is os-x only
+            sed -e '7d' ~/audius-ssh/ssh_config >> ~/.ssh/config
+      - when:
+          condition: << parameters.host >>
+          steps:
+            - run:
+                name: Perform Deployment
+                command: |
+                  cd ~/audius-protocol
+                  set -x
+                  .circleci/bin/deploy-ci.py \
+                    --github-user ${GH_USER} \
+                    --github-token ${GH_TOKEN} \
+                    --git-tag "<< parameters.git_tag >>" \
+                    --environment "<< parameters.environment >>" \
+                    --services "<< parameters.service >>" \
+                    --hosts "<< parameters.host >>" \
+                    << parameters.params >>
+      - unless:
+          condition: << parameters.host >>
+          steps:
+            - run:
+                name: Perform Deployment
+                command: |
+                  cd ~/audius-protocol
+                  set -x
+                  .circleci/bin/deploy-ci.py \
+                    --github-user ${GH_USER} \
+                    --github-token ${GH_TOKEN} \
+                    --git-tag "<< parameters.git_tag >>" \
+                    --environment "<< parameters.environment >>" \
+                    --services "<< parameters.service >>" \
+                    << parameters.params >>
+      - run:
+          name: Set Slack Detail from /tmp/summary.md
+          command: |
+            export SLACK_DETAIL=$(awk '{printf "%s\\n", $0}' /tmp/summary.md)
+            echo "export SLACK_DETAIL='${SLACK_DETAIL}'" >> "$BASH_ENV"
+      - slack-fail:
+          channel: C03STD3UXMG
+          slack_mentions_user: "@joaquin"
```

Comment:
maybe when we check this in, we should just have no user hard-coded here -- probably not great to bake this into the OSS repo

Reply:
Indeed! That's for the follow up PR and both channel and slack_mentions_user have been removed from my local working branch. I want to test the `master` branch a few times today, then I'll remove the hardcoded channel and @mention.